File: engine/engine-model/src/main/scala/tools/packet/quest/info/QuestFailure.scala
package tools.packet.quest.info

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class QuestFailure(private var _type: Byte) extends PacketInput {
  def theType: Byte = _type

  override def opcode(): SendOpcode = SendOpcode.UPDATE_QUEST_INFO
}

File: engine/engine-model/src/main/java/tools/packet/quest/info/QuestFailure.java
package tools.packet.quest.info;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record QuestFailure(Byte theType) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.UPDATE_QUEST_INFO;
   }
}